publish-ecr

```diff
@@ -50,15 +49,26 @@ while [ $# -gt 0 ]; do
       show_help
       exit 255
       ;;
-    * )
+    -* )
       >&2 echo "Unknown option $1"
       show_help
       exit 1
       ;;
+    * )
+      POSITIONAL_ARGS+=("$1")
+      ;;
   esac
   shift
 done
 
+if [ ${#POSITIONAL_ARGS[@]} -ne 1 ]; then
+  >&2 echo "Exactly one positional arg required: SOURCE_IMAGE"
+  show_help
+  exit 1
+fi
+
+SOURCE_IMAGE=${POSITIONAL_ARGS[1]}
+
 if [ -n "${PLATFORM:-}" ]; then
   if [ "$(echo "$PLATFORM" | tr "[:upper:]" "[:lower:]")" = "vrouter" ]; then
     TARGET_IMAGE_NAME=xrd/xrd-vrouter
```

Comment:
I assume it was agreed `publish-s3-bucket` does not require a help string?

Reply:
There's no args at all for that script so... I guess so?

Reply:
Preferable for `-h` or `--help` not to do anything at least

Reply:
I've added one